src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,48 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { findTabById } from '../../utils/utils';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = findTabById(tabs, selectedTabId);
+
+  const isSelectedTab = tab => tab.id === selectedTab.id;
+
+  const onTabClicked = (tab) => {
+    if (!isSelectedTab(tab)) {
+      onTabSelected(tab);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={cn({
+                'is-active': isSelectedTab(tab),
+              }) || null}
```

Comment:
```suggestion
className={cn({ 'is-active': isSelectedTab(tab) })}
```

Reply:
I disagree, it will become less readable.